nemo_curator/scripts/find_matching_ngrams.py

```diff
@@ -22,21 +22,19 @@
 from nemo_curator.utils.script_utils import ArgumentHelper
 
 
-def main(args):
-    client = get_client(**ArgumentHelper.parse_client_args(args))
+def main(args: argparse.Namespace) -> None:
+    get_client(**ArgumentHelper.parse_client_args(args))
```

Comment:
I like this a lot actually. Do you think we could do `get_client(...)` instead of `client = get_client(...)` everywhere that complains about F841?

Reply:
Haha only in this PR I had added back `client = ??` after Ruff had removed it after chatting with Vibhu about it and seems like this was missed

(See this commit https://github.com/NVIDIA/NeMo-Curator/pull/654/commits/4a80251f874e9f2e3c2dda3c8a3eeab0f892bb2c)

The rationale for adding it back was, our customers might copy code over from examples / script and we want the variable to exist in case they want to see the dashboard url etc. It's a minor reason, but for that reason we let it be.